Cargo.toml

```diff
@@ -151,7 +151,8 @@ thiserror = { version = "1.0.68" }
 toml = { version = "0.8.19" }
 trybuild = { version = "1.0.101" }
 url = { version = "2.4.1", features = ["serde"] }
-wasm-tools = { version = "1.216.0" }
+wasm-tools = { version = "1.202.0" }
+
 wit-bindgen = { version = "0.30.0" }
 
 [workspace.lints.rust]
```

Comment:
for `submodules/jco`, I suggest updating it to: https://github.com/NomicFoundation/jco/commit/a951a946df8a2e4a9db3d12d210b925a93d7ccae